spanish-translations.md

```diff
@@ -0,0 +1,19 @@
+# Automatic Spanish Translations
+
+We're happy to announce that from now on we'll deliver Spanish translations to our
+tutorials (script and subtitles). How we made this possible? You may think that
+we hired a professional who takes care of translating the content, well, let me
+proudly say that you're wrong. As programmers, we love to automate processes, so, that's
+exactly what we did. We leverage the powerful translations service of *DeepL*. DeepL is
+a translating cloud service based on an AI algorithm which is surprisingly good. But, 
+this time it wasn't as easy as making an API call, we had to heavily customize our 
+content so it supports our custom MarkDown syntax and keeping the translations as 
+accurate as possible.
+
+Since this is a process run by machines, it's likely that you'll notice "details"
```

Comment:
"misprints" maybe instead of "details"?

Reply:
Yea, maybe:

> you'll notice some "misprints" or just... silly translations. Machines haven't *quite* taken over the world yet.
> But that's ok because we've also delivered...